src/app/services/landing-page/landing-page.service.ts

```diff
@@ -89,46 +89,44 @@ export class LandingPageService implements OnDestroy {
     });
   }
 
-  public initCollections(collections: string[], configJson: Config, configMapJson: MapConfig, configId?: string, configName?: string) {
+  private initCollections(cdrs: CollectionReferenceDescription[], configJson: Config,
+    configMapJson: MapConfig, configId?: string, configName?: string) {
+    const collections = cdrs.map(c => c.collection_name);
     this.collectionService.setCollections(collections);
-    this.collectionService.getCollectionsReferenceDescription().subscribe(cdrs => {
-      this.collectionService.setCollectionsRef(cdrs);
-      const collection = (collections.find(c => c === configJson.arlas.server.collection.name));
-      if (!collection) {
-        this.spinner.hide('importconfig');
-        this.logger.error(
-          this.translate.instant('Collection ')
-          + configJson.arlas.server.collection.name
-          + this.translate.instant(' unknown. Available collections: ')
-          + collections.join(', '));
-
+    this.collectionService.setCollectionsRef(cdrs);
+    const collection = (collections.find(c => c === configJson.arlas.server.collection.name));
+    if (!collection) {
+      this.spinner.hide('importconfig');
+      this.logger.error(
+        this.translate.instant('Collection ')
```

Comment:
We should use the variables for the internationalization here instead of having the keys split

Reply:
Not the purpose of the PR